src/components/timelapse/DatePicker.vue

```diff
@@ -51,6 +58,18 @@ export default {
         formattedDates += " - " + format(dateTwo, this.dateFormat);
       }
       return formattedDates;
+    },
+    onApply() {
+      let earlyDate = new Date(this.dateOne)
+      let lateDate = new Date(this.dateTwo)
+      earlyDate = addMinutes(earlyDate, earlyDate.getTimezoneOffset()) // adjust date to user timezone
```

Comment:
So are we trying to convert `earlyDate` and `lateDate` to user's timezone here? Doesn't `new Date()` already give you the date in the user's timezone?

Reply:
Oh I see NVM